src/community/resources/index.md.njk

```diff
@@ -0,0 +1,79 @@
+---
+title: Resources
+description: Useful resources and community projects
+section: Community
+layout: layout-pane.njk
+order: 7
+---
+
+A curated list of useful resources and community projects. If you have any support requests or suggestions please contact the owner - the GOV.UK Design System team is not responsible for these resources. 
+
+
+## Create flow diagrams
+
+- [A Sketch file for creating flow diagrams of GOV.UK services](https://github.com/dashouse/govuk-design-system-flow-diagrams)⁠
+- [SVG images for creating flow diagrams of GOV.UK services](https://github.com/alirawashdeh/govuk-userflow-icons)
+
+## Create wireframes
+
+- [Balsamiq wireframes based on the GOV.UK Design System](https://github.com/enoranidi/govuk-design-system-balsamiq)⁠
+- [Sketch wireframes based on the GOV.UK Design System](https://github.com/abbott567/sketch_wireframing_kit)⁠
+
+## Use GOV.UK Frontend in different frameworks
+
+### C#
+
+- [A collection of GOV.UK Design System components built in C# / ASP.Net Core](https://github.com/cabinetoffice/govuk-design-system-dotnet)⁠
+
+
+### Node.js
+
+- [An introduction to building a hapi service using GOV.UK Frontend](https://github.com/DEFRA/hapi-govuk-examples)⁠
+
+
+### PHP
+
+- [A Drupal theme for the GOV.UK Design System](https://www.drupal.org/project/govuk_design_system)⁠
+
+
+### Python
+
+- [A Flask/Python implementation of GOV.UK Frontend](https://github.com/willp-bl/python-microservice)⁠
+- [ Classes and modules for using GOV.UK Frontend in your Jinja2-powered Python app](https://github.com/alphagov/govuk-frontend-jinja)⁠
+- [A Gulp task to convert GOV.UK Frontend Nunjucks macros to Jinja2](https://gist.github.com/andymantell/8b1bb037341f38990a4d019894abd83e)⁠
+
+
+### R
+
+- [A GOV.UK Design System theme for R Markdown](https://github.com/ukgovdatascience/govdown)⁠
+
+### Ruby
+
+- [A Rails starter app using GOV.UK Frontend](https://github.com/mec/govuk-frontend-rails)⁠
```

Comment:
I believe that [this](https://github.com/dxw/dxw_govuk_frontend_rails) is a more up-to-date version of @mec's starter template

Reply:
@peteryates yep, it's like looking at the history of me learning Rails! 

The gem has fallen behind as your work on the DfE template was much nicer, we originally used the gem on DfE Claim but switched out later to remove the dependancy, following your lead. (I think the DfE template uses webpacker now?)

I think linking to the gem would be the best move and I will try and find some time to update it and but a link in my personal repo so it all comes to a single place.

Reply:
Thanks, both of you. If you could raise a PR to update the list when you're ready, that would be much appreciated